app/services/common/generate_id.rb

```diff
@@ -0,0 +1,7 @@
+module Common
+  class GenerateId
+    def self.ccs_org_id
+      "#{Time.now.to_i}"
```

Comment:
thats what we agreed to use epoch time to generate unique integer ( MO/NK/Karishma)